gcc/rust/lex/rust-lex.h

```diff
@@ -9,6 +9,36 @@
 #include <tuple>
 
 namespace Rust {
+// Simple wrapper for FILE* that simplifies destruction.
+struct RAIIFile
+{
```

Comment:
is it worth pulling this class out into its own file? It could be handy later for reading/exporting information in the object files when we start supporting imports

Reply:
Possibly. I could do so now if you wanted, but it's basically self-contained so it could be easily done at a future date.